ksqldb-functional-tests/src/test/resources/rest-query-validation-tests/pull-queries-against-materialized-aggregates.json

```diff
@@ -1797,22 +1797,23 @@
       "statements": [
         "CREATE STREAM INPUT (ID DOUBLE KEY, IGNORED INT) WITH (kafka_topic='test_topic', value_format='JSON');",
         "CREATE TABLE AGGREGATE AS SELECT ID, COUNT(1) AS COUNT FROM INPUT WINDOW TUMBLING(SIZE 1 SECOND) GROUP BY ID;",
-        "SELECT * FROM AGGREGATE WHERE ID IN (10.1, 8.1);",
+        "SELECT * FROM AGGREGATE WHERE ID IN (10.5, 8.5);",
```

Comment:
out of interest, why did you make this change?

Reply:
Because in floating point representation in the CPU (i.e. binary), `.1` cannot be represented exactly: https://www.exploringbinary.com/why-0-point-1-does-not-exist-in-floating-point

Doing a key lookup using equality with such a value creates weird flakey results sometimes: https://floating-point-gui.de/errors/comparison/

I was banging my head against the wall with these tests not returning the row I thought it should until I tried this.  Look at the result and you'll see that the existing answer is wrong!  `0.5` can be represented exactly in binary.  With all this, it probably doesn't make sense to do `double` key lookups, but I guess we allow it.  

User beware!

Reply:
🙀